BayesianOptimization/discussion_parameters.md

```diff
@@ -0,0 +1,46 @@
+
+However, there is the option of adjusting the ES acquisition function so that we at least remove the large areas of high uncertainty. An option for this would be an evaluation of our function every 3 or 4 evaluations at the value with the highest uncertainty. In the case above this would add about 2 evaluations. That can have two effects. Either this results in a new and better maximum, or we get a nicer picture in the end. Keeping in mind that an evaluation takes an hour, one has to evaluate if this is worth the risk. Since you can always do just one more evaluation after the optimization process is done, the acquisition function should probably not be changed (if we're using ES), and depending on the result we can add more evaluations in areas we deem necessary.
+
+#### Expected Improvement
+Pairing the EI acquisition function with the RBF-kernel or the Matérn kernel with $\nu$=2.5, we get the same problem as before, that a high jump of the function is not expected and we therefore look for the maximum in the wrong places. For $\nu\in$ {0.5,1.5} we get a different behaviour. With $\nu$=0.5 the uncertainty in a small area around evaluation points is already quite high. That's why the EI tends to expect the next possible maximum closely around already evaluated points instead of looking for it elsewhere. EI tends to choose low risk low reward more than e.g. ES. Therefore the combination with $\nu$=0.5 is not recommended. The combination of EI and $\nu$=1.5 is generally working very well, because the low uncertainty around evaluated points encourages the acquisition function to explore more, but still allows some level of unsmoothness, which is necessary for high jumps. A typical plot of this combination would be the following:
+
+![](../figures/EI_matern_1.5.png)
+
+As we can see, there are no large areas with high uncertainty where there could be a hidden maximum. Also the mean function is very smooth. These are the critique points at the general recommendation, so this combination is better than that in this regard. But there are also disadvantages. For one, there are a lot of evaluations needed, about 3 more than the general recommendation. Also, the found maximum is not very accurate. That happens because of the low uncertainty around the found maximum, which tells the acquisition function, that it's not very likely that there might be a higher value there.
+
+#### Knowledge Gradient
+This acquisition function is extremely computationally expensive. Therefore this is not recommended.
+
+#### Probability of Improvement
+With every kernel this acquisition function has the same problem. It doesn't choose any risk at all, but also doesn't get any reward. It always sticks close to the highest already evaluated point and therefore doesn't find the maximum in a reasonable amount of evaluations.
```

Comment:
doesn't choose? I don't know what you mean. 
Please write does not instead of doesn't;)